modules/observability/metrics-server/base/resources/workloads/kustomization.yaml

```
@@ -0,0 +1,4 @@
+kind: Kustomization
+apiVersion: kustomize.config.k8s.io/v1beta1
```

```suggestion
apiVersion: kustomize.config.k8s.io/v1beta1
commonLabels:
  app.kubernetes.io/instance: metric-server
  app.kubernetes.io/name: metric-server
```

Can we add this two additional labels to uniform it to the other modules?